src/main/kotlin/nexters/admin/controller/hello/HelloController.kt

```diff
@@ -0,0 +1,18 @@
+package nexters.admin.controller.hello
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@Tag(name = "Hello", description = "health check 용도")
+@RestController()
+class HelloController {
+
+    @Operation(summary = "서버 살아있나?", description = "내용 없음")
+    @GetMapping("/health-check")
```

Comment:
오늘 회의 때 전체적인 문서화 컨벤션까지 이야기해보면 좋을 것 같아요.

예를 들어서 description에 "내용 없음"인 경우와 빈 문자열("")인 경우는 같은거 같은데 공존하고 있어요.

```suggestion
@Tag(name = "Hello", description = "헬스 체킹 용도")
@RestController()
class HelloController {

    @Operation(summary = "서버 살아있나?")
    @GetMapping("/health-check")
```

Reply:
오늘 회의 때 진행 못한 부분인 거 같은데ㅋㅋㅋ
토요일에 확실하게 정해보죠!

Reply:
suggestion 기능 좋네요